.secrets.baseline

```diff
@@ -0,0 +1,75 @@
+{
+  "exclude": {
+    "files": null,
+    "lines": null
+  },
+  "generated_at": "2020-11-10T09:39:42Z",
```

Comment:
Do we need `.secrets.baseline` file?

Reply:
We do if we use https://github.com/alphagov/gds-pre-commit, which agreed to use after a key was committed that shouldn’t have been.